.bazelrc

```diff
@@ -26,6 +26,9 @@ build:remote --remote_timeout=3600
 # Alt: --google_credentials=some_file.json
 build:remote --google_default_credentials=true
 
+# Improve cache hit rate
+build:remote --incompatible_strict_action_env=true
```

Comment:
huh yep, this WAS on by default at one point https://github.com/bazelbuild/bazel/issues/7026 https://github.com/bazelbuild/bazel/issues/6648